### PR TITLE
CONSOLE-4125,CONSOLE-4294: Replace SelectDeprecated component in user preferences PerspectiveDropdown

### PR DESCRIPTION
**Pull request by TheRealJon**
- Rename PerspectiveDropdown to PreferredPerspectiveSelect
- Refactor PreferredPerspectiveSelect to use supported PF Select component
- Update unit tests to use RTL

### CODE REVIEW

**Comment by TheRealJon** [comment]
QE Approver
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @reestr

File: frontend/packages/console-app/src/components/user-preferences/perspective/PreferredPerspectiveSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          property: PREFERRED_PERSPECTIVE_USER_SETTING_KEY,
+          value: selection,
+        });
+      }
+      setIsOpen(false);
+    },
+    [fireTelemetryEvent, lastViewed, preferredPerspectiveID, setPreferredPerspectiveID],
+  );
+
+  const menuToggle = React.useCallback(
```
Comment (rhamilto) [inline frontend/packages/console-app/src/components/user-preferences/perspective/PreferredPerspectiveSelect.tsx:69]: Nit:  we've been inlining these elsewhere

**Comment by yanpzhan** [comment]
Launched cluster against the pr. Regression test to Perspective Dropdown field on user preference page passed.
/label qe-approved

**Comment by TheRealJon** [comment]
@opayne1 @reestr Anything needed from me to move this forward?

**Review by opayne1 — COMMENTED** [review]
Sorry I missed this! Everything looks good to me! Just one question, is perspective switcher still the correct terminology for docs? Or would we want to start using preferred perspective in docs?

/label docs-approved

**Comment by TheRealJon** [comment]
@opayne1 Perspective switcher is still the correct terminology. This PR effects a different interaction than the main perspective switcher.

**Comment by TheRealJon** [comment]
@reestr Anything I can do to help expedite? I think this is ready to merge once you approve.

**Pull request merged**